lib/schema.js

```diff
@@ -5,14 +5,16 @@ var ajv = require('ajv')();
 var fs = P.promisifyAll(require('fs'));
 var path = require('path');
 var preq = require('preq');
+var yaml = require('js-yaml');
 
 
 /** @const */
 var SCHEMA_REPO = path.join(__dirname, '../schemas');
 
 
-function schemaPath(schema) {
-    return path.join(SCHEMA_REPO, schema.toLowerCase() + '.js');
+function schemaPath(schema, ext) {
+    ext = ext || 'js';
```

Comment:
I'd be in favor of changing this while it's still easy. @eevans, are you okay with s/.js/.json/ ?